### DISCUSSION TITLE
How to pass my own terserOptions to the latest terser-webpack-plugin coming with Webpack v5

### DISCUSSION THREAD

**Discussion by liuliangsir** [webpack/webpack#12617] [Q&A] [answered]
As the above title says, I don't know how to pass my own terserOptions to the latest terser-webpack-plugin coming with Webpack v5. After referring to webpack official documentation, I only find the information that `...` is identical to the defaults.

**Comment by sokra** [comment, marked as answer]
You can't change the options of the default plugin, but you can omit "..." and pass a new terser-webpack-plugin with new options.
